### ISSUE TITLE
Accessibility: [Inspect] The name of spinner under “PageUpDown” spinner in PrintPreview dialog should be consistent with .Net framework behavior

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#1579]
* .NET Core Version: .Net 3.0 SDK Preview8 013656 build
* Have you experienced this same bug with .NET Framework?: No

**Repro Steps:**
1.	Open attached “PrintDemo” .Net Core app in VS
[PrintDemo.zip](https://github.com/dotnet/winforms/files/3476187/PrintDemo.zip)

2.	Build and run it, click the “Preview” button to show the Print Preview dialog
3.	Open Inspect and focus on the last one: ToolStripNumericUpDown in PrintPreviewToolStrip

**Actual behavior:** 
The name of spinner under “PageUpDown” spinner  is “ToolStripNumericUpDownControl” spinner, is not same as .net framework behavior. Please see following screenshot:
![image](https://user-images.githubusercontent.com/26474449/62609968-8caa2e80-b8b7-11e9-887b-1de17c6b6841.png)

**Expected behavior:**
The name of spinner under “PageUpDown” spinner in Core should keep up with .Net framework, should be “NumericUpDown” spinner.

**Comment by merriemcgaw** [comment]
This isn't a MAS violation, but we should ensure that Accessibility Insights comes back clean. If it does we can close this. @Olina-Zhang can you confirm whether Accessibility Insights complains about this? I'm not worried about whether the name is the same as in Framework, just that it's not a MAS violation.

**Comment by Olina-Zhang** [comment]
@merriemcgaw, thanks for your confirmation! For this Spinner Name changing in .Net Core, no issue was found in Accessibility Insights. Please see following testing result:
![image](https://user-images.githubusercontent.com/26474449/62679072-2b8e6380-b968-11e9-97f6-409414218d46.png)
Closing this issue now.
